lib/pbench/cli/server/user_management.py

```diff
@@ -0,0 +1,205 @@
+import click
+
+from pbench import BadConfig
+from pbench.cli.server import config_setup, pass_cli_context
+from pbench.cli.server.options import common_options
+from pbench.server.database.database import Database
+from pbench.server.database.models.users import User
+
+
+# User create CLI
+@click.group("user_group")
+@click.version_option()
+@pass_cli_context
+def cli(ctx):
+    pass  # Entry Point
+
+
+@cli.command()
+@common_options
+@click.option(
+    "--username",
+    prompt=True,
+    required=True,
+    help="pbench server account username (will prompt if unspecified)",
+)
+@click.option(
+    "--password",
+    prompt=True,
+    hide_input=True,
+    required=True,
+    help="pbench server account password (will prompt if unspecified)",
+)
+@click.option(
+    "--email",
+    prompt=True,
+    required=True,
+    help="pbench server account email (will prompt if unspecified)",
+)
+@click.option(
+    "--first-name",
+    prompt=True,
+    required=True,
+    help="pbench server account first name (will prompt if unspecified)",
+)
+@click.option(
+    "--last-name",
+    prompt=True,
+    required=True,
+    help="pbench server account last name (will prompt if unspecified)",
+)
+@pass_cli_context
+def user_create(
+    context: object,
+    username: str,
+    password: str,
+    email: str,
+    first_name: str,
+    last_name: str,
+):
+    try:
+        # Setup the pbench server config and db access
+        config_setup(context, "pbench-user-create")
+
+        user = User(
+            username=username,
+            password=password,
+            first_name=first_name,
+            last_name=last_name,
+            email=email,
+        )
+        user.add()
+        click.echo(f"User {username} registered")
+        rv = 0
+    except Exception as exc:
+        click.echo(exc, err=True)
+        rv = 2 if isinstance(exc, BadConfig) else 1
+
+    click.get_current_context().exit(rv)
+
+
+# User delete CLI
+@cli.command()
+@common_options
+@click.argument("username")
```

Comment:
I can add the email option

Reply:
I think I suggested elsewhere that, if we accept the "thing" to be deleted as an "argument" instead of an "option", then we don't have to get too hung up on what it contains right now.  That is, it's up to the implementation, not the user, to make sense of it.  So, we could implement a series of fallbacks, such that if the user gives us a username _or_ and email _or_ a database ID, the interface would be the same and it would be up to the implementation to find the "thing" and decide whether to delete it or not.  Then the only remaining problem is how to document what it does, and we can be somewhat vague for the first cut at that.

Reply:
So, I don't think we need separate options for delete-by-username and delete-by-email (especially since "username" and "email" might well end up being the same thing).